dev-manual/api/download-do.rst

```diff
@@ -0,0 +1,53 @@
+.. _api-download-do:
+
+================================
+Download digital object endpoint
+================================
+
+**GET /api/informationobjects/<slug>/digitalobject**
+
+This endpoint will stream the content of the master :term:`digital object`
+associated with the :term:`archival description` whose slug is provided.
+
+.. NOTE::
+
+   This endpoint supports streaming locally stored online
+   :term:`digital objects <digital object>`, as well as
+   external digital objects linked via a public URI. It is not possible to
+   stream offline digital objects with this endpoint.
+
+To be able to query the endpoint, you will need to know the :term:`slug` of
+the information object (aka :term:`archival description`) whose digital object
+you wish to download. Note that the slug is included in the
+:ref:`api-browse-io` response, so you could use it to determine the slug of a
+particular description if needed.
+
+.. _download-do-ex-req:
+
+Example request
+===============
+
+.. code-block:: none
+
+   curl -H "REST-API-Key: api-key" "https://www.example.com/api/informationobjects/test-description/digitalobject" --output sample_file.pdf
```

Comment:
To get something similar to your example response below, the user needs to add the `-v` verbose flag to the CURL command.

Reply:
Thanks @peterVG, and good spot! I added the `-v` flag to the example request and added a sentence prior to the example response indicating that it has been truncated/edited for readability.